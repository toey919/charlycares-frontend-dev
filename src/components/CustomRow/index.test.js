import React from 'react';
import { shallow } from 'enzyme';
import CustomStyledRow, { CRow } from './index';

describe('CustomRow', () => {
  it('Should render styled component', () => {
    let wrapper = shallow(<CustomStyledRow />);
    expect(wrapper).toMatchSnapshot();
  });
  it('Should render component without custom props', () => {
    let wrapper = shallow(<CRow noPadding padding="20px" />);
    expect(wrapper).toMatchSnapshot();
  });
});
