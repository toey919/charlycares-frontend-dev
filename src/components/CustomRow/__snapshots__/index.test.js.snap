// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomRow Should render component without custom props 1`] = `<GridRow />`;

exports[`CustomRow Should render styled component 1`] = `
<CRow
  className="sc-bdVaJa dLnFPs"
/>
`;
